frontend/awx/resources/templates/WorkflowVisualizer/WorkflowVisualizerLoading.tsx

```diff
@@ -0,0 +1,31 @@
+import { Bullseye, EmptyState, EmptyStateHeader, Icon, Spinner } from '@patternfly/react-core';
+import { ShareAltIcon } from '@patternfly/react-icons';
+import { useTranslation } from 'react-i18next';
+import { getPatternflyColor } from '../../../../../framework';
+
+export function WorkflowVisualizerLoading() {
```

Comment:
Not my favorite name. Open to suggestions.

Reply:
Some suggestions: WorkflowVisualizerLoader, WorkflowLoader, WorkflowSpinner.